ckanext/validation/jobs.py

```diff
@@ -177,6 +177,11 @@ def _validate_table(source, _format='csv', schema=None, reference_resources=[],
     limit_errors = options.pop('limit_errors', None)
     limit_rows = options.pop('limit_rows', None)
 
+    # handle schema sync
```

Comment:
This comment says very little about the change. Could be updated to something like:
```
# handle schema_sync frictionless option that ignores header (column) ordering
```